frb_example/pure_dart/lib/src/rust/api/attribute.dart

```diff
@@ -14,7 +14,7 @@ Future<void> handleCustomizedStructTwinNormal(
     RustLib.instance.api.handleCustomizedStructTwinNormal(val: val, hint: hint);
 
 Future<UserIdTwinNormal> nextUserIdTwinNormal(
-        {required UserIdTwinNormal userId, dynamic hint}) =>
+        {UserIdTwinNormal userId = const UserIdTwinNormal(), dynamic hint}) =>
```

Comment:
nit: would be great to add a test ensuring this

Reply:
Added new fixture to test this! Also uses pretty_assertions for diffing.